emcc.py

```diff
@@ -1995,10 +1995,7 @@ def get_clang_command_asm(src_file):
         # output the dependency rule. Warning: clang and gcc behave differently
         # with -MF! (clang seems to not recognize it)
         logger.debug(('just preprocessor ' if has_dash_E else 'just dependencies: ') + ' '.join(cmd))
-        shared.print_compiler_stage(cmd)
-        rtn = run_process(cmd, check=False).returncode
-        if rtn:
-          return rtn
```

Comment:
This previous code seems to ignore if `run_process` exists with a nonzero exit code, but the new code will call `sys.exit(1)` (inside `check_call`) in case of an error. Is this change intended?

Reply:
Returning non-zero from this function (main) is that same as exiting the whole process with non-zero.   So I'm pretty sure there is no change here, just a cleanup.